spec/features/login_spec.rb

```diff
@@ -30,3 +30,57 @@
     end
   end
 end
+
+describe 'login via email/password' do
+  before :each do
+    visit '/'
+    expect(page).to have_content 'Please login first'
```

Comment:
Naja, das ist halt das, was wir im Slack schon diskutiert hatten. Ich dachte da auch mehr an Konsistenz als an Funktionalität. Aber OK, wenn du es so lassen möchtest ist es natürlich auch in Ordnung, funktionieren tut es ja :)